src/libraries/System.Net.Sockets/src/System/Net/Sockets/AcceptOverlappedAsyncResult.Unix.cs

```diff
@@ -14,9 +14,9 @@ namespace System.Net.Sockets
     // AcceptOverlappedAsyncResult - used to take care of storage for async Socket BeginAccept call.
     internal sealed partial class AcceptOverlappedAsyncResult : BaseOverlappedAsyncResult
     {
-        private Socket _acceptedSocket;
+        private Socket? _acceptedSocket;
 
-        internal Socket AcceptSocket
+        internal Socket? AcceptSocket
```

Comment:
NIT: not sure why its need to be nullable

Reply:
The one place that sets this property sets it to a `Socket?`. And on Unix it is always `null`.

https://github.com/dotnet/runtime/blob/03fc3ab4f63e6bbb99f300769b786e72007d98ff/src/libraries/System.Net.Sockets/src/System/Net/Sockets/Socket.cs#L3504